### DISCUSSION TITLE
TP 4 Ejercicio 8 - Busqueda: Segmentation fault

### DISCUSSION THREAD

**Discussion by LautaroUlloa** [INGCOM-UNRN-P1/cursada-2023#139] [Preguntas y respuestas] [answered]
Buenas, estuve tratando de hacer el código del ejercicio 8, llegué a esto pero cuando lo ejecuto me dice "Segmentation fault". Alguien tiene idea de por qué podría ser?

https://github.com/INGCOM-UNRN-P1/tp4-2023-LautaroUlloa

**Comment by martinvilu** [comment, marked as answer]
Un `Segmentation fault` es un acceso de memoria fuera de lugar, un `arreglo[10000]`, por ejemplo

**Reply by martinvilu** [reply]
Cargá el código en el repositorio.

**Reply by LautaroUlloa** [reply]
Ya lo cargué

**Reply by LautaroUlloa** [reply]
Ya logré resolver los errores. Muchas gracias Martín
